packages/wrangler/src/__tests__/mock-confirm.ts

```diff
@@ -0,0 +1,29 @@
+import { confirm } from "../dialogs";
+
+/**
+ * The expected values for a confirmation request.
+ */
+export interface ConfirmExpectation {
```

Comment:
does this need to be exported?

Reply:
I am not sure if it "has" to be exported. But since it is used as the type of a parameter of an exported function it seems the right thing to do...

Reply:
Oh, is this because if someone wants to construct the arguments separately before passing it to mockConfirm/mockPrompt, they'll have the type to use? Ok, I can buy that.